### PR TITLE
feat(header): implement header

### PR DESCRIPTION
**Pull request by mthanh2209**

### CODE REVIEW

File: src/assets/css/components/button.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.btn {
+    color: var(--primary-color);
+    font-family: var(--primary-font);
+    font-size: 16px;
+    font-weight: var(--fw-semibold);
+  }
+  .btn--secondary {
```
Comment (hungnguyenhuu-agilityio) [inline src/assets/css/components/button.css]: Check the indent

File: src/assets/css/components/button.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    padding: 10px 22.22px 9px 25.25px;
```
Comment (hungnguyenhuu-agilityio) [inline src/assets/css/components/button.css]: The number is magic. Should check

File: src/assets/css/layouts/header.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
+.header {
```
Comment (hungnguyenhuu-agilityio) [inline src/assets/css/layouts/header.css]: Naming

File: src/assets/css/components/button.css (hunk 1 above)
Comment (mthanh2209) [reply src/assets/css/components/button.css]: thank you, i fixed it

File: src/index.html [hunk 4]
```diff
@@ -6,6 +6,35 @@
     <link rel="stylesheet" href="assets/css/main.css" />
     <title>Figma Land</title>
   </head>
-
-  <body></body>
+  <style>
+    body {
+      background: #1f2e35;
+    }
+  </style>
```
Comment (hungnguyenhuu-agilityio) [inline src/index.html]: Why do you set the style on HTML file

File: src/assets/css/components/button.css [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    border: 1px solid #bcd0e5;
+    text-align: center;
+  }
+  .btn--primary {
+    padding: 10px 23px 9px 25px;
+    text-decoration: none;
+    flex-shrink: 0;
+    border-radius: 4px;
+    background: #02897a;
+  }
```
Comment (hungnguyenhuu-agilityio) [inline src/assets/css/components/button.css]: With the button, should split into another PR

File: src/assets/css/components/button.css (hunk 2 above)
Comment (mthanh2209) [reply src/assets/css/components/button.css]: Thank you, On figma I look up the numbers like that, I don't know if I round like this it's still oke?
`padding: 10px 23px 9px 25px;`

File: src/assets/css/layouts/header.css (hunk 3 above)
Comment (mthanh2209) [reply src/assets/css/layouts/header.css]: Thank you, I fixed it

File: src/index.html (hunk 4 above)
Comment (mthanh2209) [reply src/index.html]: Thank you, i moved it to variable.css

File: src/assets/css/components/button.css (hunk 5 above)
Comment (mthanh2209) [reply src/assets/css/components/button.css]: Thank you, I will create a new branch and PR it

File: src/assets/css/components/button.css [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  border: 1px solid #bcd0e5;
+  text-align: center;
+}
+.btn--primary {
+  padding: 10px 23px 9px 25px;
+  text-decoration: none;
+  flex-shrink: 0;
+  border-radius: 4px;
+  background: #02897a;
+}
```
Comment (hungnguyenhuu-agilityio) [inline src/assets/css/components/button.css]: This one is on another PR. Pls remove

File: src/assets/css/layouts/header.css [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+  align-items: center;
+}
+.header_logo {
+  padding-right: 20px;
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  text-decoration: none;
+}
+.logo_img {
```
Comment (hungnguyenhuu-agilityio) [inline src/assets/css/layouts/header.css]: Naming

File: src/assets/css/layouts/header.css [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+  justify-content: space-between;
+}
+.header_list {
+  list-style: none;
+  display: inherit;
+  align-items: center;
+  margin-right: 20px;
+  line-height: 28px;
+}
+.header_list li {
```
Comment (hungnguyenhuu-agilityio) [inline src/assets/css/layouts/header.css]: Should be item. Should not use element like that

File: src/index.html [hunk 9]
```diff
@@ -7,5 +7,37 @@
     <title>Figma Land</title>
   </head>
 
-  <body></body>
+  <body>
+    <!-- START HEADER SECTION -->
+    <header class="header_container">
+      <a href="#" class="header_logo">
```
Comment (hungnguyenhuu-agilityio) [inline src/index.html]: Using javascript:void(0). Check it here https://stackoverflow.com/questions/1291942/what-does-javascriptvoid0-mean. And update the similar cases related

File: src/assets/css/components/button.css (hunk 6 above)
Comment (mthanh2209) [reply src/assets/css/components/button.css]: sorry, i removed it

File: src/index.html (hunk 9 above)
Comment (mthanh2209) [reply src/index.html]: Thank you, i updated it

File: src/assets/css/layouts/header.css (hunk 7 above)
Comment (mthanh2209) [reply src/assets/css/layouts/header.css]: Thank you, I fixed it

File: src/assets/css/layouts/header.css (hunk 8 above)
Comment (mthanh2209) [reply src/assets/css/layouts/header.css]: Thank you, i fixed it

File: src/index.html [hunk 10]
```diff
@@ -7,5 +7,32 @@
...
+    <!-- START HEADER SECTION -->
+    <header class="header_container">
+      <a href="javascript:void(0)" class="header_logo">
+        <img src="assets/images/logo.png" alt="Logo" class="header_logo_img" />
+        <span class="header_logo_text">Product</span>
+      </a>
+      <nav class="header_navbar">
+        <ul class="header_list">
+          <li>
+            <a href="javascript:void(0)" class="header_link" style="font-weight: var(--fw-semibold)">Product</a>
```
Comment (hungnguyenhuu-agilityio) [inline src/index.html]: Why you use the styles directly in the HTML ? Should update

Comment (mthanh2209) [reply src/index.html]: Thank you, i updated it

File: src/assets/css/layouts/header.css [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+}
+.header_logo {
+  font-weight: var(--fw-bold);
+  padding-right: 20px;
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+.header_logo_img {
+  padding: 19px 0 19px 146px;
```
Comment (hungnguyenhuu-agilityio) [inline src/assets/css/layouts/header.css]: The numbers are magic

File: src/assets/css/layouts/header.css [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+.header_logo_text {
+  padding: 29px 0px;
+  font-size: 30px;
+}
+.header_navbar {
+  margin-right: 271px;
+  display: flex;
+  justify-content: space-between;
+}
+.header_list {
```
Comment (hungnguyenhuu-agilityio) [inline src/assets/css/layouts/header.css]: header_navbar_item will be better

File: src/assets/css/layouts/header.css (hunk 11 above)
Comment (mthanh2209) [reply src/assets/css/layouts/header.css]: Thank you, I fixed it

File: src/assets/css/layouts/header.css (hunk 12 above)
Comment (mthanh2209) [reply src/assets/css/layouts/header.css]: thank you
